CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
  ### React Addons
   
-* Remove PropTypes dependency from ReactLink. ([@gaearon](https://github.com/gaearon) in [#9766](https://github.com/facebook/react/pull/9766))
-* Fix issue where environment variable was not being transformed by browserify. ([@mridgway](https://github.com/mridgway) in [#9642](https://github.com/facebook/react/pull/9642))
```

Comment:
I'm guessing the reason for removing these two items is that they are both relatively small fixes? I could see that making sense, if that is your intention.